cmd/eksctl-anywhere/cmd/installpackagecontroller.go

```diff
@@ -72,7 +72,7 @@ func installPackageController(ctx context.Context) error {
 	}
 
 	curatedpackages.PrintLicense()
-	err = ctrlClient.EnableCuratedPackages(ctx)
+	err = ctrlClient.EnableCuratedPackages(ctx, &clusterSpec.Cluster.Spec)
```

Comment:
I think injecting this into ctrlClient will make this so much easier.

Reply:
Done.